scripts/tools.sh

```diff
@@ -33,7 +33,7 @@ fi
 # air - code changes watcher
 if ! command -v air &> /dev/null
 then
-  curl -sSfL https://raw.githubusercontent.com/cosmtrek/air/master/install.sh | sh -s -- -b $(go env GOPATH)/bin
+  go install github.com/cosmtrek/air@latest
```

Comment:
Buildne sa to zo zdrojakov, nie je to velky program, tak tam nie je velky rozdiel.